### DISCUSSION TITLE
如何取消control + period 的键位绑定

### DISCUSSION THREAD

**Discussion by yayuwang0624** [rime/home#1057] [Q&A]
我注意到任意情况下control + peroid  会激活emoji输入，这跟我其他的键位冲突
但是在default.yaml设置中搜索 `control + peroid` 并没有绑定在相关功能上，即使设置`- {accept: "Control+Period", send: noop, when: always}` 也会激活emoji输入，请问应该在哪里设置这一键位呢？

**Comment by LEOYoon-Tsaw** [comment]
你激活的繪文字輸入不是Rime的吧

**Reply by yayuwang0624** [reply]
哈哈，研究了半天没想到这个问题，居然是ibus自带的，感谢
